src/rules/style.ts

```diff
@@ -36,7 +38,7 @@ module.exports = {
   // Disallows parameter properties in class constructors.
   'no-parameter-properties': false,
   // Enforces consistent object literal property quote style.
-  'object-literal-key-quotes': [true, 'as-needed'],
+  'object-literal-key-quotes': [true, 'always'],
```

Comment:
Ahh, thank you so much for catching this! :+1:
I forgot to revert this when I was doing some testing.

This rule gets annoying when you have a huge object literal like this file. For one word key's it suggests that you don't use quotes. 
Similar to what we have in [`eslint-plugin-shopify`](https://github.com/Shopify/eslint-plugin-shopify/blob/master/lib/config/.eslintrc.js#L3), I've overrode this rule within this repo in [`tslint.json`](ab014f3). 

@GoodForOneFare, is this okay?